Consensus.md

```diff
@@ -1,48 +1,324 @@
 more than 1001/1000, and cannot adjust downwards by more than 999/1000.
 
-If there are not 2000 blocks, the genesis timestamp is used as the moment to
-compare to. The expected time is 10 minutes * (block height).
-
-------------------
--- Transactions --
-------------------
-
-A transaction is valid if:
-	+ All inputs are fully signed, without frivilous signatures
-	+ All inputs spend known outputs
-	+ The spend conditions in each input match the spend hash of the output
-		being spent.
-	+ The inputs equal the outputs
-		- Outputs can be outputs
-		- Miner Fees can be outputs
-		- File Contracts can be outputs
-	+ All contracts have a non-zero payout
-	+ All storage proofs act on contracts found in the blockchain
-	+ All storage proofs are correct
-
-The output created by a storage proof is neither considered an input or an
+The new target is calculated using (expected time passed in seconds) / (actual
+time passed in seconds) * (current target). The division and multiplication
+should be done using infinite precision, and the result should be truncated.
+
+If there are not 2000 blocks, the genesis timestamp is used for comparison.
+The expected time is (10 minutes * block height).
+
+The difficulty clamp means that the target can shift by at most 7.5x in 2016
+blocks, which can be compared to the 4x clamp of Bitcoin. The amount of work
+required to quadruple the difficulty in Sia is 3000x the starting difficulty,
+which can be compared to 2016x for Bitcoin. The amount of work required to 16x
+the difficulty in Sia is 15,000x the original difficulty, which can be compared
+to 10,000x for Bitcoin.
+
+Block Subsidy
+-------------
+
+The coinbase for a block is (300,000 - (1 * height)) * 2^80, with a minimum of
+30,000 * 2^80. Any miner fees get added to the coinbase, which creates the
+block subsidy. The block subsidy is then given to multiple outputs, called the
+miner outputs. The total value of the miner outputs must equal the block
+subsidy.  Having multiple outputs allows the block reward to be sent to
+multiple people, enabling systems like p2pool.
+
+The outputs created by the block subsidy cannot be spent for 100 blocks, and
+are not considered a part of the utxo set until 100 blocks have transpired.
+This limitation is in place because a simple reorg is enough to invalidate the
+output; double spend attacks and false spend attacks are much easier.
+
+Transactions
+------------
+
+A Transaction is composed of a set of inputs, miner fees, outputs, file
+contracts, storage proofs, arbitrary data, and signatures. The sum of the
+inputs must equal the sum of the miner fees, outputs, and contract payouts.
+
+A Transaction is composed of the following:
+- Inputs
+- Miner Fees
+- Outputs
+- File Contracts
+- Storage Proofs
+- Siafund Inputs
+- Siafund Outputs
+- Arbitrary Data
+- Signatures
+
+The financial sum of all the inputs must equal the financial sum of all the
+miner fees, outputs, and contract payouts. There can be no leftovers.
```

Comment:
"financial" is probably redundant here. At most I would use "monetary," and only for the first instance (the rest being implied).